app/data/1_0001.json

```diff
@@ -0,0 +1,5134 @@
+                            "id": "innovation-investment-section-25",
+                            "number": "3",
+                            "questions": [
+                                {
+                                    "answers": [],
+                                    "description": "",
+                                    "id": "innovation-investment-question-25",
+                                    "title": "",
+                                    "type": "General"
+                                }
+                            ],
+                            "title": "Innovation Investment"
+                        }
+                    ],
+                    "type": "interstitial"
+                }
+            ],
+            "id": "innovation-investment-section-25-group-25",
+            "title": "Innovation Investment"
+        },
+        {
+            "blocks": [
+                {
+                    "id": "goods-and-services-innovation-block-26",
+                    "sections": [
+                        {
+                            "description": "<p>3 year period 1 January 2014 to 31 December 2016.</p>",
+                            "id": "goods-and-services-innovation-section-26",
+                            "number": "4",
+                            "questions": [
+                                {
+                                    "answers": [
+                                        {
+                                            "description": "",
+                                            "id": "goods-and-services-innovation-answer-26-0",
+                                            "label": "",
+                                            "mandatory": false,
+                                            "options": [
+                                                {
+                                                    "label": "Yes",
+                                                    "value": "Yes"
+                                                },
+                                                {
+                                                    "label": "No",
+                                                    "value": "No"
+                                                }
+                                            ],
+                                            "q_code": "0510",
+                                            "type": "Radio"
+                                        }
+                                    ],
+          
```

Comment:
This description should be within another guidance block so that it appears after the Exclude list
![image](https://cloud.githubusercontent.com/assets/1754822/21856898/70723282-d81b-11e6-901d-1b6c770fc698.png)